workflows/VGP-assembly-v2/README.md

```diff
@@ -0,0 +1,28 @@
+# Vertebrate Genome Project in Galaxy
+
+## VGP pipeline
+
+The VGP pipeline is composed of:
+    
+- Five main workflows
+
+1. Quality control and parameter estimation. It provides a kmer coverage database and an estimation of parameters that will be useful in the genome assembly process.
```

Comment:
Is this the workflow in workflows/VGP-assembly-v2/VGP-meryldb-creation-trio or workflows/VGP-assembly-v2/VGP-meryldb-creation? How do you decide which workflow to use? If you have paternal and maternal data you would use the trio workflow, and if not you would use the non-trio version?

Reply:
Yes, exactly. Results have shown that by using the trios approach the problems associated with incorrect assignation of reads to primary/alternate assemblies are reduced quite a lot.